platform/platform.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/trustwallet/blockatlas/platform/zilliqa"
 )
 
+const emptyParam = ""
```

Comment:
Any reason it’s called this way? If it’s an emptyString

Reply:
Param to init Collection API, gives a bit more details about behavior of function from my point of view